kafka-ui-react-app/src/components/Topics/New/__test__/New.spec.tsx

```diff
@@ -79,12 +79,12 @@ describe('New', () => {
       userEvent.click(screen.getByText(/submit/i));
     });
 
```

Comment:
why should new Page tests suites change if we just added Route for copy , maybe the switch gets redirected to the wrong thing.